client/asset/dcr/dcr.go

```diff
@@ -1647,3 +1653,11 @@ func fees(tx *wire.MsgTx) (uint64, float64) {
 	fees := in - out
 	return uint64(fees), float64(fees) / float64(tx.SerializeSize())
 }
+
+// isTxNotFoundErr will return true if the error indicates that the requested
+// transaction is not known.
+func isTxNotFoundErr(err error) bool {
+	// TODO: Could probably do this right with errors.As if we enforce an RPC
+	// version when connecting.
+	return strings.HasPrefix(err.Error(), "-5:")
```

Comment:
Checking for a `"-:5:"` prefix is just as fragile as checking against `dcjrson.ErrRPCNoTxInfo` (a const set to -5), so let's use the code check.